crates/turborepo-lockfiles/src/berry/mod.rs

```diff
@@ -249,7 +259,7 @@ impl BerryLockfile {
 
     /// Produces a new lockfile containing only the given workspaces and
     /// packages
-    pub fn subgraph(
+    fn subgraph(
```

Comment:
Make this private to ensure external callers go through the trait and don't call this directly.